src/components/Table/Table.js

```diff
@@ -130,10 +130,11 @@ export const TableUser = ({ ensName, address }: TableUserProps) => {
 
 export const TableFee = ({ txFeeInWei, gasToken, chain }: TableFeeProps) => {
   const decimals = gasToken?.decimals || nativeAssetPerChain[chain].decimals;
+  const gasTokenAddress = gasToken?.address || nativeAssetPerChain[chain].address;
   const formattedFee = txFeeInWei ? formatUnits(txFeeInWei.toString(), decimals) : '0';
   const gasSymbol = getGasSymbol(chain, gasToken);
 
-  return <TableAmount amount={formattedFee} token={gasSymbol} chain={chain} />;
+  return <TableAmount amount={formattedFee} assetSymbol={gasSymbol} assetAddress={gasTokenAddress} chain={chain} />;
```

Comment:
Consider dropping assetSymbol prop in favor of TableAmount quiering supported assets for it in order to simplify TableAmount usage.

Reply:
Per our internal discussion added this as TODO: : get assetSymbol from matching asset once assets can be queried by assetAddress as key instead of performing expensive search on whole assets array.